src/sed.js

```diff
@@ -22,6 +22,9 @@ function _sed(options, regex, replacement, files) {
     'i': 'inplace'
   });
 
+  // Check if this is coming from a pipe
+  var pipe = common.readFromPipe(this);
```

Comment:
Idea: have a 'pipe' option for common.wrap, when enabled pass pipe as the last arg. (Optional)